evm_scilla_js_tests/test/BlockProperties.ts

```diff
@@ -0,0 +1,65 @@
+import {expect} from "chai";
+import hre from "hardhat";
+import {Contract} from "ethers";
+import {ethers} from "hardhat";
+
+describe("Block Properties", function () {
+  let contract: Contract;
+  before(async function () {
+    contract = await hre.deployContract("BlockAndTransactionProperties");
+  });
+
+  it("Should be deployed successfully @block-1", async function () {
+    expect(contract.address).to.be.properAddress;
+  });
+
+  it("should return the correct block number", async function () {
+    const contractBlockNumber = await contract.getBlockNumber();
+    const currentBlockNumber = await ethers.provider.getBlockNumber();
+
+    expect(contractBlockNumber).to.equal(currentBlockNumber);
+  });
+
+  it("should return the correct block timestamp", async function () {
+    const contractBlockTimestamp = await contract.getTimestamp();
+    const currentBlock = await ethers.provider.getBlock("latest");
```

Comment:
Shouldn't we specify a fixed block number in all of these tests and pass the same one for both the `eth_call` and the `getBlock`? Otherwise, the network might move forward a block between these 2 lines causing flaky tests?